api/aws/dynamodb/v1/dynamodb.proto

```diff
@@ -151,6 +157,28 @@ message Throughput {
   int64 read_capacity_units = 2;
 }
 
+// Represents the Key structure for the table
+message KeySchema {
+  enum Type {
+    // Undetermined value
+    UNSPECIFIED = 0;
```

Comment:
you should add UNKNOWN for the case when the type cannot be determined coming from the AWS SDK.

Reply:
Ok, will do!